cmd/config.go

```diff
@@ -564,15 +564,16 @@ func (c *Config) ValidatePathEnd(pe *relayer.PathEnd) error {
 		return err
 	}
 
+	chain, err := c.Chains.Get(pe.ChainID)
+	if err != nil {
+		return err
+	}
+
 	// if the identifiers are empty, don't do any validation
 	if pe.ClientID == "" && pe.ConnectionID == "" && pe.ChannelID == "" {
 		return nil
 	}
 
-	chain, err := c.Chains.Get(pe.ChainID)
```

Comment:
is this change intentional? The previous check is cheaper, and prevents a panic if the user hasn't added the chain but also hasn't set path end information (and therefore doesn't need validation)

Reply:
Yeah, this is intentional. When given client, connection and channel identifiers are empty, it will return nil and path will be created. But we are not checking chains are configured or not. This also throwing error `chain not configured` error when querying paths.

Reply:
I'm confused. Do any configuration checks occur if the client, connection, and channel id are all empty? It is my understanding that if all the identifiers are empty, there is no configuration validation that needs to be done. Setting all empty identifiers is a valid starting place on the relayer

Reply:
Yeah, Okay. But can chain-id be empty as we need to set chain before setting path?

Reply:
Ahh, chain ID should be set. Can you just add a check that pe.ChainID != "" as the first if statement in this function? This way we return an error instead of panicing